docs/sources/fundamentals/intro-to-prometheus/index.md

```diff
@@ -0,0 +1,110 @@
+
+{{< figure src="/media/docs/grafana/intro-prometheus/rate-function.png" max-width="750px" caption="Prometheus rate function" >}}
+
+A counter metric is just one type of metric. Prometheus [supports several others](https://prometheus.io/docs/concepts/metric_types/). A counter is a number (such as total bytes written) that only goes up. A second metric type is called a `gauge`, which can go up or down.
+
+The following gauge visualization shows the total RAM usage on a computer.
+
+{{< figure src="/media/docs/grafana/intro-prometheus/gauge-example.png" max-width="750px" caption="Gauge visualization" >}}
+
+The third metric type is called a `histogram`, which counts observations and puts them into configurable groups. The following example shows floating-point numbers grouped into ranges that show how often each occurred.
+
+{{< figure src="/media/docs/grafana/intro-prometheus/histogram-example.png" max-width="750px" caption="Historgram visualization" >}}
+
+These core concepts of time series, metrics, labels, and aggregation functions are foundational to Grafana and observability.
+
+## Why this is valuable
+
+Software and systems are a difficult business. Sometimes things go wrong. Observability helps you understand a system’s state so that future issues can be proactively addressed. And when problems do occur, you can diagnose and solve them within your Service Level Objectives (SLOs). The [three pillars of observability](https://www.oreilly.com/library/view/distributed-systems-observability/9781492033431/ch04.html) are metrics, logs, and traces and Prometheus supports the metric pillar. When software on a computer runs slowly, observability can help identify whether the CPU is saturated, the system is out of memory, or if the disk is writing at maximum speed.
+
+## Prometheus as software
+
+Prometheus isn’t just a data format; many people refer to it as an [open source systems monitoring and alerting toolkit](https://prometheus.io/docs/introduction/overview/). That’s because Prometheus is software, not just data.
+
+Prometheus can scrape metric data from software and infrastructure and store it. Scraping means that Prometheus software periodically revisits the same endpoint to check for new data. Prometheus “scrapes” data from a piece of software that is instrumented with a client library.
+
+For example, a NodeJS application can configure the [prom-client](https://github.com/siimon/prom-client) to expose metrics easily at an endpoint, and Prometheus can regularly scrape that endpoint. There are a number of other tools available within the Prometheus toolkit that you can use to instrument applications.
+
+## Prometheus as deployment
```

Comment:
we see to use _italics_ and "quotes" interchangeably for the same concepts here. "Prometheus as Data/Software" and _Promethtus as Data/Software_. I would suggest not using either and just capitalizing Data and Software, or just picking one formatting option

Reply:
Good call. thanks!